lib/node_modules/@stdlib/assert/is-constantcase/bin/cli

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
+
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var readFileSync = require( '@stdlib/fs/read-file' ).sync;
+var CLI = require( '@stdlib/cli/ctor' );
+var stdin = require( '@stdlib/process/read-stdin' );
+var stdinStream = require( '@stdlib/streams/node/stdin' );
+var isConstantCase = require( './../lib' );
+
+
+// MAIN //
+
+/**
+* Main execution sequence.
+*
+* @private
+* @returns {void}
+*/
+function main() {
+	var flags;
+	var opts;
+	var cli;
+
+	// Create a command-line interface:
+	cli = new CLI({
+		'pkg': require( './../package.json' ),
+		'options': require( './../etc/cli_opts.json' ),
+		'help': readFileSync( resolve( __dirname, '..', 'docs', 'usage.txt' ), {
+			'encoding': 'utf8'
+		})
+	});
+
+	// Get any provided command-line options:
+	flags = cli.flags();
+	if ( flags.help || flags.version ) {
+		return;
+	}
+
+	// Convert the provided command-line option to a corresponding option value:
+	opts = {
+		'sep': flags.split
+	};
+
+	// Check if we are receiving data from `stdin`...
+	if ( !stdinStream.isTTY ) {
+		stdin( onRead );
+		return;
+	}
+	if ( cli.args.length ) {
+		console.log( isConstantCase( cli.args[ 0 ] ) ); // eslint-disable-line no-console
+	} else {
+		console.log( isConstantCase() ); // eslint-disable-line no-console
+	}
+
+	/**
+	* Callback invoked upon reading from `stdin`.
+	*
+	* @private
+	* @param {(Error|null)} error - error object
+	* @param {Buffer} data - data
+	* @returns {void}
+	*/
+	function onRead( error, data ) {
+		var str;
+		var i;
+		if ( error ) {
+			return cli.error( error );
+		}
+		str = data.toString();
+		if ( opts.sep ) {
+			str = str.split( opts.sep );
+			for ( i = 0; i < str.length; i++ ) {
+				console.log( isConstantCase( str[ i ] ) ); // eslint-disable-line no-console
+			}
+		} else {
+			console.log( isConstantCase( str ) ); // eslint-disable-line no-console
+		}
```

Comment:
This logic is incorrect. See https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/assert/is-kebabcase/bin/cli#L80 for the correct logic.

cc @Planeshifter

Reply:
oh okay, I actually will need to rewrite the tests.

Reply:
The tests should be fine, unless you changed them. It is just the logic in the CLI file which needs updating.

Reply:
oh okay, I was not able to push those changes after I rebase it over my current PR, it threw me error, hence I had to change those.